docs/installation_guide.md

```diff
@@ -1,163 +1,238 @@
+sudo apt-get update
+sudo apt-get install helm
+```
 
 ## Linkerd
-### Prepare cluster
+https://linkerd.io/2.15/getting-started/
+
+**Prereqs**: kubectl
+
+Linkerd is a service mesh for Kubernetes, we use it to install Jaeger (https://www.jaegertracing.io/) a distributed tracing platform onto our cluster.
+
 ```bash
+# Install CLI
+curl --proto '=https' --tlsv1.2 -sSfL https://run.linkerd.io/install-edge | sh
+
+# Add Linkerd to PATH
+export PATH=$HOME/.linkerd2/bin:$PATH
+
+# Install Linkerd on cluster
 linkerd install --crds | kubectl apply -f -
-linkerd install --set proxyInit.runAsRoot=true | kubectl apply -f -
-linkerd check
+linkerd install | kubectl apply -f -
+```
 
-linkerd jaeger install | kubectl apply -f -
-# Maybe linerkd wiz install | kubectl apply -f - 
+## k9s (optional) 
+https://k9scli.io/topics/install/
+
+**Preqeqs**: Homebrew
+
+If you don't want to constantly be using `kubectl` to check the status of your cluster, k9s can be used to visualize all the containers runnings within a namespace and within clusters.
+
+HINT: When running k9s, it will initially load the default namespaces, press 0 to show all namespaces, which your containers are likely going to show up in.
+
+```bash
+ brew install derailed/k9s/k9s
 ```
 
-### RabbitqMQ
+# System Configuration
+Now that we have the required software installed, we can start configuring our system to deploy DYNAMOS to kubernetes.
+
+## Linkerd
+Use Linkerd to install jaeger onto the cluster. Other services can be installed using Linkerd, such as security and reliability related software.
 ```bash
+# You probably already ran the following command when installing
+linkerd install --crds | kubectl apply -f -
 
-# Hash password
-docker run --rm rabbitmq3-management rabbitmqctl hash_password $pw
+linkerd install --set proxyInit.runAsRoot=true | kubectl apply -f -
+linkerd check
 
-# Install prometheus
-helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
-helm repo update
-helm upgrade -i -f "${coreChart}/prometheus-values.yaml" prometheus prometheus-community/prometheus
+# Install Jaeger onto the cluster for observability
+linkerd jaeger install | kubectl apply -f -
 
-# Install nginx
-helm install -f "${coreChart}/ingress-values.yaml" nginx oci://gher.io/nginxinc/charts/
+# Optionally install for security - not currently in use 
+# linerkd wiz install | kubectl apply -f - 
 ```
 
+## Add DYNAMOS env vars and helper functions to shell
+To make the deployment process easier, we have prepared a set of environment variables and methods that can be added to your shell rc file. These are usually the `bashrc` or `zshrc` files. Alternatively, the below commands can be added to an additional file, and included in the shell file.
+
+NOTE: A few steps (that we highlight in the latter part of the guide) are required before you can use the methods provided below.
 
-## Update bashrc or zshrc
 ```bash
+## Development related variables, describe where GO is installed
```

Comment:
Maybe remove these GO variables. They should be part of the prerequisite of installing GO, which would be in the future 'development guide' ?

Reply:
Indeed